jekyll/_cci1/index.md

```diff
@@ -5,39 +5,39 @@ title: "CircleCI 1.0 Documentation"
-{% endif %}
-
-<div class="category-section">
-	<img src="{{ site.baseurl }}/assets/img/icons/{{ category.icon }}" class="logo" alt="Category Icon" />
-	<h2>{{ category.name }}</h2>
-	<ul class="list-unstyled">
-	{% assign docs_found = 0 %}
-	{% assign sortedDocs = site.cci1 | sort: 'order' %}
-	{% for doc in sortedDocs %}
-		{% if doc.categories contains category.slug and doc.slug != catFile and doc.hide != true %}
-			{% assign docs_found = docs_found | plus: 1 %}
-			{% if docs_found < 4 %}
-				{% if doc.short-title %}
-					<li class="{% if page.path contains doc.url %}active{% endif %}"><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.short-title }}</a></li>
-				{% else %}
-					<li><a href="{{ site.baseurl }}{{ doc.url }}">{{ doc.title }}</a></li>
-				{% endif %}
-			{% endif %}
-		{% endif %}
-	{% endfor %}
-
-	{% if docs_found > 3 %}
-		{% assign more = docs_found | minus: 3 %}
-		<li><em><a href="{{ site.baseurl }}/1.0/{{ catFile }}/">{{ more }} more</a></em></li>
-	{% endif %}
-	</ul>
-</div>
-{% endif %}
-{% endfor %}
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The hosted solution is available at <https://circleci.com/>. The [CircleCI Enterprise](https://circleci.com/enterprise/) solution is installable inside your private cloud or data center. Both are free to try for two weeks. If you are new to CircleCI, check out the [Getting Started with CircleCI]({{ site.baseurl }}/1.0/getting-started/).
+
+## Overview
+
+CircleCI automates build, test, and deployment of software for mobile, enterprise, and web applications. 
+
+![CircleCI Example Flow with GitHub]({{ site.baseurl }}/assets/img/docs/how_it_works.png)
+
+For example, after a software repository on GitHub or Bitbucket is authorized and added as a project to the circleci.com SaaS application, every new commit triggers a build and notification of success or failure through webhooks. CircleCI also supports Slack, HipChat, Campfire, Flowdock, and IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
+
+## Programming Language Support
+
+For some language versions, CircleCI provides demo applications, as follows:  
+
+- **Go 1.5.3**, see the [Golang Guide]({{ site.baseurl }}/1.0/language-go/)
+- **Haskell 7.4 and later**, see the [Haskell Guide]({{ site.baseurl }}/1.0/language-haskell/)
+- **Java JDK6 and later**, see the [Java Guide]({{ site.baseurl }}/1.0/language-java/)
+- **Node.js all versions**, see the [Node Guide]({{ site.baseurl }}/1.0/language-nodejs/)
+- **PHP 5 and later**, see the [PHP Guide]({{ site.baseurl }}/1.0/language-php/)
+- **Python 2.6.6 and later**, see the [Python Guide]({{ site.baseurl }}/1.0/language-python/)
+- **Ruby 1.8 and later**, see the [Ruby/Rails Guide]({{ site.baseurl }}/1.0/language-ruby-on-rails/) 
+- **Scala 0.13.11 and later**, see the [Scala Guide]({{ site.baseurl }}/1.0/language-scala/)
+
```

Comment:
(I'm uncomfortable with how the version numbers are stated for 2.0 on https://circleci.com/docs/2.0/ - it's not accurate since other versions are possible if you build a Docker image with the version you want. Also developers wouldn't phrase it like 'Python 2 and above' since noone writes Python 1.0 code. The statement seems redundant. We should state that we support building for all language version on 2.0 - that's the flexibility you get with Docker)